content/en/docs/ops/ambient/usage/waypoint/index.md

```diff
@@ -0,0 +1,400 @@
+TODO
+
+## Verify the waypoint proxy is deployed
+
+{{< text bash >}}
+$ code for verification
+{{< /text >}}
+
+This indicates the L7 waypoint proxy is working.  In the next section we look at how to monitor the confuguration and data plane of the L7 waypoint proxy to confirm that traffic is correctly using the L7 waypoint proxy. 
+
+### Verify that the Waypoint proxy is routing traffic to the application
+
+## Configuring the Waypoint Proxy - 
+
+### Overview
+
+This section describes how to configure the waypoint proxy for the Bookinfo application. The Bookinfo application is a sample application that requires a virtual service to route traffic to its different services.
+
+The core functionality of the waypoint L7 traffic management is identical to sidecar mode, hence to add more features refer to this link: https://istio.io/latest/docs/reference/config/networking/virtual-service/
+
+FOR EACH SECTION:
+
+### Creating a Virtual Service
+
+To create a virtual service for the Bookinfo application, you can use the following YAML manifest:
+
+{{< text bash >}}
+$ yaml file
+{{< /text >}}
+
+This virtual service will route all traffic to the `bookinfo-v1` service.
+
+### Deploying the Virtual Service
+
+To deploy the virtual service, you can use the following command:
+
+{{< text bash >}}
+$ command
+{{< /text >}}
+
+### Verifying the Waypoint Proxy Configuration is working
+
+Once the virtual service is deployed, you can verify the waypoint proxy configuration by running the following command:
+
+{{< text bash >}}
+$ command
+{{< /text >}}
+
+This will output the configuration of the waypoint proxy, including the virtual service that is mapped to it.
+
+### Configuring Virtual Services
+
+You can configure both L4 and L7 virtual services for waypoint proxies. If you want to do a Virtual Service with TCP that is effectively a L4 virtual services. This is used to route traffic to services based on port number. L7 virtual services are used to route traffic to services based on more complex criteria, such as HTTP method and path. In a Virtual service you can have only TCP, or only HTTP or Both.
+
+<< Considered merging L4 (TCP) and L7 (HTTP) virtual service >>
+
+### Example
+
+The following YAML manifest shows an example of an L7 virtual service:
+
+{{< text bash >}}
+$ yaml file
+{{< /text >}}
+
+This virtual service will route traffic to the `bookinfo-v1` service for requests to the `/productpage` path and traffic to the `bookinfo-v2` service for requests to the `/reviews` path.
+
+### Verifying the Virtual Service
+
+Once the virtual service is set up, the HTTP route is mapped to the waypoint configuration. This means that all traffic that matches the virtual service's hosts and HTTP routes will be routed to the waypoint proxy.
+
+You can verify this by using the `istioctl proxy-config` command to dump the waypoint configuration. This will show you the virtual services that are mapped to the waypoint proxy.
```

Comment:
"You can verify this by using the `istioctl proxy-config` command to dump the waypoint configuration. This will show you the virtual services that are mapped to the waypoint proxy."

Might want to change to something like:
In classic mode you can use `istioctl proxy-config cmds` to dump the envoy configuration. While `istioctl proxy-config cmds` will still work in ambient there are some differences because the envoy proxy is no longer configured for every sidecar.    

Would then follow the section about the istioctl proxy-config cmds found here https://istio.io/latest/blog/2023/waypoint-proxy-made-simple/